well-architected/sustainability/sustainability-design-methodology.md

```diff
@@ -119,6 +119,11 @@ Therefore, building a carbon score dashboard can make use of the following proxi
 - Usage over time (requests, users, API calls, etc.)
 - Any extra measurement that is relevant to the application
 
+Further examples of implementing this can be found on the following pages:
+
+- [DataOps - Measure Azure App Sustainability](https://learn.microsoft.com/en-us/azure/architecture/example-scenario/apps/measure-azure-app-sustainability-sci-score)
+- [ISE Developer Blog - Saving CO2 using location and time shifting in Azure](https://devblogs.microsoft.com/ise/saving-co2-using-location-and-time-shifting-in-azure/)
```

Comment:
```suggestion
```
Removing this suggestion, as the link is a bit outdated now and doesn't mention Azure carbon optimization, which gives you resource-insights to carbon emissions; something that blog post mentions isn't possible at the time of writing.